tests/redirects/map_globalconf.py

```diff
@@ -1050,10 +1050,6 @@
 
     # Bug 1255785
     url_test('/styleguide/identity/mozilla/logo-prototype/', '/styleguide/'),
-    # Bug 1268847
-    url_test('/styleguide/websites/sandstone/buttons/', '/styleguide/'),
```

Comment:
I don't think there's any reason to remove these redirect tests, as they are for tabzilla content that's already been removed?